backend/game/cards/keep_cards/health_manipulation_cards/even_bigger.py

```diff
@@ -0,0 +1,20 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+
+
+class EvenBigger(KeepCard):
+    def __init__(self):
+        super(KeepCard, self).__init__("Even Bigger", 4,
+                                         "Your maximum [Heart] is increased by 2. Gain 2[Heart] when you get this card",
+                                         "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
```

Comment:
This is a bit confusing of a function as you define a method in the parent class, then overload it, but use the parent's original method in the body of child?  I think this could be refactored

Reply:
was attempting to reduce code duplication, however your right. I was hiding too much. Coupling to self of sorts. Much harder to maintain.

Added player.add_card() and player.remove_card() card calls to immediate_effect and discard_effect methods respectively to fix.

Updated test_even_bigger.py